src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -189,7 +195,7 @@ export default class AddStaffNotationDialog extends Vue {
 
   /** Whether this filing is an Administrative Dissolution. */
   get isAdministrativeDissolution (): boolean {
-    return this.isTypeAdministrativeDissolution({ name: this.name, dissolutionType: this.dissolutionType })
+    return EnumUtilities.isTypeDissolutionAdministrative({ filingSubType: this.dissolutionType })
```

Comment:
I updated `isTypeDissolutionAdministrative` (and others) to work for different object types, so I was able to simplify this code (and other places).